global/data/example-data.json

```diff
@@ -4,9 +4,7 @@
         "1": {
```

Comment:
Some more fields that can be removed from both JSON files, if set to an empty/null value:
- all boolean variables which are set to false, if not required. (Maybe they are all required? Then it is fine)
- `meeting`: `custom_translations`, `start_time`, `end_time`
- `mediafile/pdf_information`
- `projection/options`
- empty collections in `example_data.json`: `motion_statute_paragraph`, `chat_message`
- all empty collections in `initial_data.json`

Also make sure to remove all unnecessary fields from the import tests.

Reply:
Done.